### ISSUE TITLE
Integrate tox.ini and mxdev

### ISSUE THREAD

**Issue by ericof** [plone/meta#151]
How to add another source package (implemented by mxdev) to tox.ini?

**Comment by ericof** [comment]
A possible workaround would be:

- [ ] Allow, via configuration, to set `constrain_package_deps` to False (See this i[ssue in Tox](https://github.com/tox-dev/tox/issues/2897))
- [ ] Set `constraints-mxdev.txt` on the option `constraints_file` for tox
- [ ] Allow, in `[testenv:test]` and `[testenv:coverage]`, additional `deps`,

**Comment by ericof** [comment]
@gforcada It was a bit more complex than I estimated, but I believe it is doable. See the changes in the [`portalbrasil`](https://github.com/plone/meta/tree/portalbrasil) branch

**Comment by gforcada** [comment]
@jensens I know `mxdev` and `mxmake` cover this and more, but if we can, specially for single repositories I would try to defer adding yet another layer of abstraction as much as possible...

Our stack is complex enough, but in general, only for a few cases we need some extra super powers (mxdev and such).

@ericof the `tox -e init` sounds like a nice approach 👍🏾  I think that at least one or two options can be maybe simplified, but I would have to test myself.

Should we document/configure `mxdev.ini` as well, or is out of scope for `plone/meta` ? 🤔

**Comment by gforcada** [comment]
@ericof the branch looks good, some love and polishing might be needed, but overall looks great 👍🏾 are you expecting to do more changes before we can get a PR and get it reviewed/merge? 🤔

**Comment by ericof** [comment]
I just need to get back and do a bit of clean-up. I will do it until Thursday

**Comment by jensens** [comment]
> Should we document/configure mxdev.ini as well, or is out of scope for plone/meta ? 

As soon as one (like me) uses plone/meta for collective packages or customer projects in its own repos having  `mx.ini` (it is used for mxmake too) covered is a must have.

**Comment by gforcada** [comment]
Yes, indeed, the question is if we should also explain how to use it, set it up, etc. or expect that everyone already knows.

Maybe pointing to a few examples, whenever they are configured might be a fine balance between not documenting, but giving examples at your fingertips 👍🏾
